### PR TITLE
Condense Ghost Planchette equipment effects

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/equipment-effects/effect-ghostcallers-planchette-armor.json [hunk 1]
```diff
@@ -21,13 +21,61 @@
...
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.EquipmentVariety",
+                "rollOption": "ghostcallers-planchette"
+            },
             {
                 "exceptions": [
                     "ghost-touch"
                 ],
                 "key": "Resistance",
+                "predicate": [
```
Comment (simonward) [inline packs/equipment-effects/effect-ghostcallers-planchette-armor.json:63]: I wonder if we should have a custom exception to handle "attacks by incorporeal creatures count as ghost touch"?

Comment (ammalagonc) [reply packs/equipment-effects/effect-ghostcallers-planchette-armor.json:63]: Probably- how do we set that up?

Comment (simonward) [reply packs/equipment-effects/effect-ghostcallers-planchette-armor.json:63]: Take a look at Vampire Count. `PF2E.IWR.Custom.MagicalSilver`

File: static/lang/re-en.json [hunk 2]
```diff
@@ -684,6 +684,7 @@
                 "EnemyStrikes": "enemy Strikes",
                 "HolyBludgeoningPiercingSlashing": "holy bludgeoning, piercing and slashing damage",
                 "ImpulseFireFrom": "impulse fire from {origin}",
+                "IncorporealCreatures": "incorporeal creatuters",
```
Comment (simonward) [inline static/lang/re-en.json:687]:
```suggestion
                "IncorporealCreatures": "incorporeal creatures",
```

File: packs/equipment-effects/effect-ghostcallers-planchette-weapon.json [hunk 3]
```diff
@@ -44,8 +78,33 @@
                 ],
                 "key": "AdjustStrike",
                 "mode": "add",
+                "predicate": [
```
Comment (simonward) [inline packs/equipment-effects/effect-ghostcallers-planchette-weapon.json:81]: Is this predicate needed?

File: packs/equipment-effects/effect-ghostcallers-planchette-armor.json [hunk 4]
```diff
@@ -21,13 +21,93 @@
...
+                    "ghostcallers-planchette:ghostcallers-planchette"
+                ],
                 "type": "physical",
                 "value": 5
+            },
+            {
+                "exceptions": [
```
Comment (simonward) [inline packs/equipment-effects/effect-ghostcallers-planchette-armor.json:70]: Probably need to have both exceptions in the resistances?

File: packs/equipment-effects/effect-ghostcallers-planchette-armor.json [hunk 5]
```diff
@@ -21,13 +21,105 @@
...
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.EquipmentVariety",
+                "rollOption": "ghostcallers-planchette"
+            },
             {
                 "exceptions": [
                     "ghost-touch"
                 ],
                 "key": "Resistance",
+                "predicate": [
```
Comment (simonward) [inline packs/equipment-effects/effect-ghostcallers-planchette-armor.json:63]: This Resistance RE should probably be removed?

File: packs/equipment-effects/effect-ghostcallers-planchette-armor.json [hunk 6]
```diff
@@ -21,13 +21,105 @@
...
+                ],
                 "type": "physical",
                 "value": 5
+            },
+            {
+                "exceptions": [
+                    "ghost-touch"
+                ],
+                "key": "Resistance",
+                "predicate": [
```
Comment (simonward) [inline packs/equipment-effects/effect-ghostcallers-planchette-armor.json:74]: And this one?

File: packs/equipment-effects/effect-ghostcallers-planchette-armor.json [hunk 7]
```diff
@@ -21,13 +21,105 @@
...
+                    "ghostcallers-planchette:ghostcallers-planchette-greater"
+                ],
+                "type": "physical",
+                "value": 10
+            },
+            {
+                "exceptions": [
+                    {
+                        "definition": [
+                            "item:trait:magical",
```
Comment (simonward) [inline packs/equipment-effects/effect-ghostcallers-planchette-armor.json:84]: Not sure we need magical?

File: packs/equipment-effects/effect-ghostcallers-planchette-armor.json [hunk 8]
```diff
@@ -21,13 +21,105 @@
...
+                            {
+                                "or": [
+                                    "item:rune:property:ghost-touch",
```
Comment (simonward) [inline packs/equipment-effects/effect-ghostcallers-planchette-armor.json:87]: Can probably move ghost touch to top of exceptions?

File: packs/equipment-effects/effect-ghostcallers-planchette-armor.json [hunk 9]
```diff
@@ -21,13 +21,81 @@
...
+                    }
+                ],
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.EquipmentVariety",
+                "rollOption": "ghostcallers-planchette"
+            },
             {
                 "exceptions": [
-                    "ghost-touch"
+                    {
```
Comment (simonward) [inline packs/equipment-effects/effect-ghostcallers-planchette-armor.json:60]: For your exceptions, try:
```
"exceptions": [
    "ghost-touch",
    {
      "definition": [
        "origin:trait:incorporeal"
      ],
      "label": "PF2E.IWR.Custom.IncorporealCreatures"
    }
  ]
```

Comment (ammalagonc) [reply packs/equipment-effects/effect-ghostcallers-planchette-armor.json:60]: Changes made

File: packs/equipment-effects/effect-ghostcallers-planchette-armor.json [hunk 10]
```diff
@@ -21,13 +21,81 @@
...
+                        "definition": [
+                            "ghost-touch",
+                            {
+                                "or": [
```
Comment (simonward) [inline packs/equipment-effects/effect-ghostcallers-planchette-armor.json:64]: This "or" isn't needed - just the inner clause.

File: packs/equipment-effects/effect-ghostcallers-planchette-armor.json [hunk 11]
```diff
@@ -21,13 +21,81 @@
...
                 "type": "physical",
                 "value": 5
+            },
+            {
+                "exceptions": [
+                    {
+                        "definition": [
+                            "ghost-touch",
+                            {
+                                "or": [
```
Comment (simonward) [inline packs/equipment-effects/effect-ghostcallers-planchette-armor.json:85]: Same here.

**Pull request merged**